release/src/Util.hs

```diff
@@ -380,6 +380,7 @@ isReleaseCommit :: MonadCI m => m Bool
 isReleaseCommit = do
     files <- gitChangedFiles "HEAD"
     let isRelease = "VERSION" `elem` files
+                 && "unreleased.rst" `elem` files
                  && "docs/source/support/release-notes.rst" `elem` files
                  && length files == 2
```

Comment:
```suggestion
                  && length files == 3
```

I think ^

Reply:
& maybe change the exception below to mention `unreleased.rst` as well

Reply:
Ooop, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1762/commits/7574d179921f8de806e3cfed0c2568f454c9eacb

Reply:
Perfect, thank you!